content/docs/localization/joining-the-localization-team.md

```diff
@@ -1,46 +1,115 @@
 ---
-title: Joining the Localization Team
+title: Help translate Scratch Addons

```

Comment:
The title should be kept to be in line with other pages of the docs

Reply:
https://github.com/ScratchAddons/website-v2/pull/363/commits/9e8bcd65bb5390308e2a055a48a68633ffbf986b